support/mock/mockrt.go

```diff
@@ -8,14 +8,14 @@ import (
 	"testing"
 
 	addr "github.com/filecoin-project/go-address"
-	cid "github.com/ipfs/go-cid"
+	"github.com/ipfs/go-cid"
```

Comment:
Please retain the aliases, at least where they don't match the path. See https://github.com/golang/go/issues/28428